docs/Tutorials/CCE.md

```diff
@@ -0,0 +1,243 @@
+\cond NEVER
+Distributed under the MIT License.
+See LICENSE.txt for details.
+\endcond
+# %Running CCE {#tutorial_cce}
+
+The basic instructions for getting up and running with a stand-alone
+CCE using external data are:
+- Clone spectre and build the CharacteristicExtract target
+- *Note*: CCE currently requires the linked HDF5 library to be built
+  thread-safe. As this is a significant restriction on many systems,
+  we hope to eliminate this requirement in the future.
+- At this point (provided the build succeeds) you should have the
+  executable `bin/CharacteristicExtract` in the build directory; You can now
+  run it using an input file to provide options. The input file
+  `tests/InputFiles/Cce/CharacteristicExtract.yaml` from the spectre
+  source tree can help you get started with writing your input file.
+  There are a few important notes there:
+  - For resolution, the example input file has lmax (`Cce.LMax`) of 12, and
+    filter lmax (`Filtering.FilterLMax`) of 10; that'll run pretty fast but
+    might be a little low for full precision. lmax 16, filter 14 should be
+    pretty good, and typically precision doesn't improve above lmax 24,
+    filter 22 (be sure to update the filter as you update lmax -- it should
+    generally be around 2 lower than the maximum l to reduce possible aliasing).
+  - If you want to just run through the end of the provided worldtube data,
+    you can just omit the `EndTime` option and the executable will figure it
+    out from the worldtube file.
+  - The `ScriOutputDensity` adds extra interpolation points to the output,
+    which is useful for finite-difference derivatives on the output data, but
+    otherwise it'll just unnecessarily inflate the output files, so if you
+    don't need the extra points, best just set it to 1.
+  - If you're extracting at 100M or less, best to reduce the `TargetStepSize`,
+    to around .5 at 100M and lower yet for nearer extraction.
+  - The `InitializeJ` in the example file uses `InverseCubic` which is a pretty
+    primitive scheme, but early tests indicate that it gives the best results
+    for most systems.
+    If initial data is a concern, you can also try replacing the `InverseCubic`
+    entry with:
+    ```
```

Comment:
this did not render correctly for me (chrome on my macbook pro) I had to indent the triple ticks only two spaces